appinventor/components/tests/com/google/appinventor/components/runtime/RegressionTest.java

```diff
@@ -0,0 +1,51 @@
+package com.google.appinventor.components.runtime;
+
+
+import com.google.appinventor.components.common.LOBFValues;
+import com.google.appinventor.components.runtime.util.YailList;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class RegressionTest extends RobolectricTestBase {
+  Regression regression = new Regression(getForm());
+  LOBFValues value;
+  YailList xList = YailList.makeList(Arrays.asList("1", "2", "3","4","5","6"));
+  YailList yList = YailList.makeList(Arrays.asList("1", "2", "3","2","2","88"));
```

Comment:
Move the initialization into the `setUp()` method and make sure to call `super.setUp()`.